### PR TITLE
1099 rmc 1663 handle qid linking ccs

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
https://trello.com/c/KNsTMSMf/1099-rmc-1663-handle-qid-linking-in-ccspropertylisted-event-5

# What has changed
New test to check if passing a qid with the a CCSCaseListed results in no msg being sent to fwmt

# How to test?
Install https://github.com/ONSdigital/census-rm-case-processor/tree/1099-css-with-qid then run the tests

# Links
https://trello.com/c/KNsTMSMf/1099-rmc-1663-handle-qid-linking-in-ccspropertylisted-event-5

# Screenshots (if appropriate):

### CODE REVIEW

**Review by AdeHarrison — COMMENTED** [review]
return of mysterious unused sample_file.csv file

File: acceptance_tests/features/steps/ccs_property_listed.py [hunk 1]
```diff
@@ -58,6 +59,52 @@ def send_ccs_property_listed_event(context):
...
+                    "latitude": "50.863849",
+                    "longitude": "-1.229710",
+                    "fieldcoordinatorId": "XXXXXXXXXX",
+                    "fieldofficerId": "XXXXXXXXXXXXX"
+                },
+                "uac": {
+                    "questionnaireId": context.expected_questionnaire_id
+                }
+            }
+        }
```
Comment (AdeHarrison) [inline acceptance_tests/features/steps/ccs_property_listed.py]: loads of duplicate code and will only get worse, make this a common method adding the props we want for each test?

File: acceptance_tests/features/ccs_property_listed.feature [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
     Then the CCS Property Listed case is created
-    And the correct ActionInstruction is sent to FWMT
\ No newline at end of file
+    Then the correct ActionInstruction is sent to FWMT
+
+  Scenario: Log event when a CCS Property Listed event is received with a qid
+    When an unaddressed message of questionnaire type 71 is sent
+    Then a UACUpdated message not linked to a case is emitted to RH and Action Scheduler
+    When a CCS Property Listed event is sent with a qid
+    Then the CCS Property Listed case is created
```
Comment (AdeHarrison) [inline acceptance_tests/features/ccs_property_listed.feature:13]: is this an "and"?

**Review by AdeHarrison — APPROVED** [review]

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Stray 59000 line sample file included?

File: acceptance_tests/features/ccs_property_listed.feature [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
 
-  Scenario: Log event when a CCS Property Listed event is received
+  Scenario: Log event when a CCS Property Listed event is received with no qid
     When a CCS Property Listed event is sent
     Then the CCS Property Listed case is created
     And the correct ActionInstruction is sent to FWMT
     And the case API returns the CCS QID for the new case
+
+  Scenario: Log event when a CCS Property Listed event is received with a qid
+    When an unaddressed message of questionnaire type 71 is sent
```
Comment (AdamHawtin) [inline acceptance_tests/features/ccs_property_listed.feature]:
```suggestion
    Given an unaddressed message of questionnaire type 71 is sent
```

File: acceptance_tests/features/ccs_property_listed.feature [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
+    And a UACUpdated message not linked to a case is emitted to RH and Action Scheduler
+    When a CCS Property Listed event is sent with a qid
+    And the CCS Property Listed case is created
```
Comment (AdamHawtin) [inline acceptance_tests/features/ccs_property_listed.feature]:
```suggestion
    Then the CCS Property Listed case is created
```

**Review by AdamHawtin — DISMISSED** [review]

**Comment by LukeLoze1** [comment]
> Stray 59000 line sample file included?

I'm guessing this was on a previous branch somewhere, will get it out

**Review by ryangrundy7 — APPROVED** [review]
All tests pass 👍

**Pull request merged**